src/toxcore/messenger/packet/mod.rs

```diff
@@ -99,7 +99,7 @@ impl FromBytes for Packet {
 
 #[cfg(test)]
 mod tests {
-    use super::*;
+    use super::{Packet, *};
```

Comment:
What was wrong with previous `use`?

Reply:
```
error[E0599]: no variant named `Online` found for type `toxcore::messenger::packet::group::Packet` in the current scope
   --> src/toxcore/messenger/packet/mod.rs:106:17
    |
106 |         Packet::Online(Online)
    |         --------^^^^^^
    |         |
    |         variant not found in `toxcore::messenger::packet::group::Packet`
    | 
   ::: src/toxcore/messenger/packet/group/mod.rs:96:1
    |
96  | pub enum Packet {
    | --------------- variant `Online` not found here
```

It conflicts with other `Packet` type.

Reply:
Then with `pub use self::group::*;` we have 2 conflicting `Packet` definitions. And it seems one of them can't be imported outside this crate. So we either shouldn't use `pub use` or should rename one of these packets (with qualified import, for example).

Reply:
done